### PR TITLE
bumping ospray to 1.7.3

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

File: bbp/viz/embree/default.nix [hunk 1]
```diff
@@ -1,11 +1,12 @@
 { stdenv
 , fetchFromGitHub
 , cmake
+, pkgconfig
 , ispc
 , tbb
-, freeglut 
+, freeglut
```
Comment (karjonas) [inline bbp/viz/embree/default.nix]: I don't think you need this if you do not build the tutorials.

File: bbp/viz/ospray/default.nix [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 stdenv.mkDerivation rec {
 	name = "ospray-${version}";
-	version = "1.5.0";
+	version = "1.7.1";
```
Comment (tribal-tec) [inline bbp/viz/ospray/default.nix]: I would vote for 1.7.3

**Comment by ppodhajski** [comment]
is Brayns compiled with clang supposed to work with ospray compiled with intel compiler? I get a nasty segfault

**Comment by karjonas** [comment]
I see no reason why not

File: bbp/viz/embree/default.nix [hunk 3]
```diff
@@ -1,11 +1,11 @@
 { stdenv
 , fetchFromGitHub
 , cmake
+, pkgconfig
 , ispc
 , tbb
-, freeglut 
 , mesa
```
Comment (karjonas) [inline bbp/viz/embree/default.nix]: I think you do not need any of the deps below anymore

**Comment by karjonas** [comment]
I think you should rename this PR since it is more about bumping to latest OSPRay.

File: bbp/viz/ospray/default.nix [hunk 4]
```diff
@@ -24,8 +24,8 @@ stdenv.mkDerivation rec {
 	src = fetchFromGitHub {
 		owner = "Bluebrain";
```
Comment (tribal-tec) [inline bbp/viz/ospray/default.nix]: not using the offical upstream?

**Pull request merged**